app/controllers/physical_network_port_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PhysicalNetworkPortController < ApplicationController
+  include Mixins::GenericShowMixin
+  include Mixins::GenericListMixin
+
+  before_action :check_privileges
+  before_action :session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def self.model
+    @model ||= "PhysicalNetworkPort".safe_constantize
+  end
+
+  def title
+    _('Network Ports')
+  end
+
+  def model
+    self.class.model
+  end
+
+  def self.table_name
+    @table_name ||= "physical_network_ports"
+  end
+
+  def session_data
```

Comment:
Please, use the GenericSessionMixin if possible. I will make future cleanups simpler if we stick the the same code.

Reply:
Oh, cool! Will do! :)